infrastructure/server-setup/tasks/updates.yml

```diff
@@ -1,23 +1,15 @@
 - name: Update apt repository
   apt:
     update_cache: yes
+    force_apt_get: yes
 
-- name: Upgrade all packages to the latest version
-  apt:
-    upgrade: dist
-
-- name: Remove unnecessary packages
-  apt:
-    autoremove: yes
-
-- name: Clean up apt cache
-  apt:
-    autoclean: yes
-
-- name: Install unattended-upgrades package
-  apt:
-    name: unattended-upgrades
-    state: present
+- name: Manage apt updates, upgrades, and package installation
+  shell: |
+    while fuser /var/lib/dpkg/lock >/dev/null 2>&1; do sleep 5; done
+    apt-get update && apt-get dist-upgrade -y
+    apt-get install -y unattended-upgrades
+    apt-get autoremove -y
+    apt-get autoclean -y
```

Comment:
@n1koo can you take a look at this. My main objective was to get these so that the operations wouldn't fail even if you would run apt operations to a machine from two different pipeline at the same time. 

If you currently try to do that, you'll get this error

```
fatal: [backup]: FAILED! => {"changed": false, "msg": "'/usr/bin/apt-get dist-upgrade ' failed: E: Could not get lock /var/lib/dpkg/lock-frontend. It is held by process 137120 (apt-get)\nE: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), is another process using it?\n", "rc": 100, "stdout": "", "stdout_lines": []}
```

https://github.com/opencrvs/opencrvs-euanland/actions/runs/9483734935/job/26131561246#step:11:77